app/cafe.py

```diff
@@ -0,0 +1,20 @@
+import datetime
+
+from app.errors import (NotVaccinatedError, OutdatedVaccineError,
```

Comment:
I think it's better to use indentation according to the checklist example

Reply:
agree, fix this